lib/CMakeLists.txt

```diff
@@ -10,3 +10,14 @@ FetchContent_Declare(argon
 )
 FetchContent_MakeAvailable(argon)
 
+add_compile_definitions(NDEBUG)
+set(NE10_ASM_OPTIMIZATION ON)
+set(NE10_TARGET_ARCH armv7)
+
+FetchContent_Declare(Ne10
+  GIT_REPOSITORY https://github.com/stellar-aria/Ne10
```

Comment:
Is it possible to pin a specific version or commit?

Reply:
done!